content/docs/tasks/telemetry/distributed-tracing-lightstep/index.md

```diff
@@ -0,0 +1,143 @@
+        --set global.tracer.lightstep.accessToken="<access-token>" \
+        --set global.tracer.lightstep.secure=true \
+        --set global.tracer.lightstep.cacertPath="/etc/lightstep/cacert.pem" \
+        install/kubernetes/helm/istio \
+        --name istio --namespace istio-system > $HOME/istio.yaml
+    $ kubectl create namespace istio-system
+    $ kubectl apply -f $HOME/istio.yaml
+    {{< /text >}}
+
+1.  Store your satellite pool's certificate authority certificate as a secret in the default namespace.
+    If the Bookinfo application will be deployed in a different namespace, create the secret in that namespace instead.
+
+    {{< text bash >}}
+    $ CACERT=$(cat Cert_Auth.crt | base64) # Cert_Auth.crt contains the necessary CACert
+    $ NAMESPACE=default
+    {{< /text >}}
+
+    ```bash
+    $ cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: lightstep.cacert
+        namespace: $NAMESPACE
+        labels:
+          app: lightstep
+      type: Opaque
+      data:
+        cacert.pem: $CACERT
+    EOF
+    ```
+
+1.   Deploy the Bookinfo sample application by following [these instructions](/docs/examples/bookinfo/#deploying-the-application).
+
+## Visualize trace data
+
+1.  Create an ingress gateway for the Bookinfo application by following
+    [these instructions](/docs/examples/bookinfo/#determining-the-ingress-ip-and-port).
+
+1.  Verify the previous step's success by confirming that the environment variable `GATEWAY_URL` is
+    populated in your shell.
+
+1.  Send traffic to the sample application.
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Load the LightStep [𝑥]PM web UI.
+
+1.  Navigate to Explorer.
+
+1.  Find the query bar at the top. The query bar allows you to interactively filter results by a Service, Operation, and/or Tag values.
+
+1.  Select `productpage.default` from the Service drop-down list.
+
+1.  Click **Run**. You see something similar to the following:
+
+    {{< image width="100%" ratio="50%"
+    link="./istio-tracing-list-lightstep.png"
+    caption="Explorer"
+    >}}
+
+1.  Click on the first row in the table of example traces below the latency histogram to see the details
+    corresponding to your refresh of the `/productpage`. The page then looks similar to:
+
+    {{< image width="100%" ratio="50%"
+    link="./istio-tracing-details-lightstep.png"
+    caption="Detailed Trace View"
+    >}}
+
+The screenshot shows that the trace is comprised of a set of spans. Each span corresponds to a Bookinfo service invoked
+during the execution of a `/productpage` request.
+
+Two spans in the trace represent every RPC. For example, the call from `productpage` to `reviews` starts
+with the span labeled with the `reviews.default.svc.cluster.local:9080/*` operation and the
+`productpage.default: proxy client` service, which represents the client-side span of the call. The screenshot shows
```

Comment:
Ambiguity due to compound subject of the dependent clause. Basically, readers cannot determine whether it is the service, the operation, or both that represent the client-side of the call. The entire sentence, beginning in line 116 and ending here is a long complex run-on sentence. I suggest splitting it:
```suggestion
`productpage.default: proxy client` service. This service represents the client-side span of the call. The screenshot shows
```